float64 p_out
float64 v_out
float64 t_out
int32 can_id